src/content/docs/dropins/payment-services/containers/apple-pay.mdx

```diff
@@ -16,28 +16,58 @@ The `ApplePay` container provides the following configuration options:
-    ['isVirtualCart', 'boolean', 'Yes', 'Whether the cart contains only virtual/downloadable products.'],
+    ['location', 'string', 'Yes', 'Location where the Apple Pay button is to be rendered. Must be either "CHECKOUT" or "PRODUCT_DETAIL".'],
+    ['getCartId', 'function', 'Maybe', 'Required if createCart not provided. Should return a promise that resolves to the shopper\'s cart ID.'],
+    ['createCart', 'object', 'Maybe', 'Required if getCartId not provided. Should have a getCartItems function that returns a list with the cart items to purchase. The list must contain at least one cart item.'],
     ['onButtonClick', 'function', 'No', 'Called when the user clicks the Apple Pay button. The callback receives a "showPaymentSheet" function as its only argument that must be called to begin the Apple Pay session and show the payment sheet. If no "onButtonClick" handler is provided, the Apple Pay session will begin and the payment sheet will show on click, always.'],
     ['onSuccess', 'function', 'No', 'Called when the payment flow finishes successfully. If the function returns a promise, the promise will be awaited before marking the payment as "completed successfully" in the Apple Pay sheet. If the promise rejects, the payment will be marked as "failed" in the Apple Pay sheet and the error will be propagated to the "onError" callback, if provided.'],
     ['onError', 'function', 'No', 'Called when the payment flow was aborted due to an error. The function receives a single argument, {message: string}, containing the reason of error.'],
   ]}
 />
 
-## Example
+## Checkout page example
 
 ```js
 import ApplePay from '@dropins/storefront-payment-services/containers/ApplePay.js';
 import { render as PaymentServices } from '@dropins/storefront-payment-services/render.js';
 import * as orderApi from '@dropins/storefront-order/api.js';
+import { PaymentLocation } from '@dropins/storefront-payment-services/api.js';
 
 const cart = events.lastPayload('checkout/initialized');
 
 if (cart) {
   const $content = document.createElement('div');
   PaymentServices.render(ApplePay, {
+    location: PaymentLocation.CHECKOUT,
     getCartId: async () => cart.id,
-    isVirtualCart: cart.isVirtual,
+    onSuccess: () => orderApi.placeOrder(cart.id),
+    onError: (error) => { console.error(error) },
+  })($content);
+}
+```
+
+## Product details page example
+
+```js
+import ApplePay from '@dropins/storefront-payment-services/containers/ApplePay.js';
+import { render as PaymentServices } from '@dropins/storefront-payment-services/render.js';
+import * as orderApi from '@dropins/storefront-order/api.js';
+import { PaymentLocation } from '@dropins/storefront-payment-services/api.js';
+
+const cart = events.lastPayload('pdp/values');
+
+if (cart) {
+  const $content = document.createElement('div');
+  PaymentServices.render(ApplePay, {
+    location: PaymentLocation.PRODUCT_DETAIL,
+    createCart: {
+    getCartItems: async () => ([},
```

Comment:
Two things
1. should be indented
2. should be sync (now it returns a promise that resolves to a list. it should return a list instead)